vendor/flow.exe

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env node
```

Comment:
I don't think Windows has shebang support.

Anyway, Windows support is new enough that we can probably just delete this file and no one will notice